src/components/attendee/SearchOverlay/EventItem/EventItem.tsx

```diff
@@ -0,0 +1,94 @@
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { Button } from "components/attendee/Button";
+import { NavOverlayTabType } from "components/attendee/NavOverlay/NavOverlay";
+import { differenceInCalendarDays } from "date-fns";
+
+import { SPACE_TAXON, STRING_DASH_SPACE, STRING_SPACE } from "settings";
+
+import { ScheduledEvent } from "types/venues";
+
+import { eventEndTime, eventStartTime } from "utils/event";
+import {
+  formatDateRelativeToNow,
+  formatTimeLocalised,
+  getDayMonthNow,
+} from "utils/time";
+import { enterSpace } from "utils/url";
+
+import { useSpaceById } from "hooks/spaces/useSpaceById";
+import { useWorldById } from "hooks/worlds/useWorldById";
+
+import CN from "../SearchOverlay.module.scss";
+
+export type EventItemProps = {
+  event: ScheduledEvent;
+  onClick: () => void;
+  setNavMenu: React.Dispatch<React.SetStateAction<string | undefined>>;
+};
+
+export const EventItem: React.FC<EventItemProps> = ({
+  event,
+  onClick,
+  setNavMenu,
+}) => {
+  const { space } = useSpaceById({ spaceId: event.spaceId });
+  const { world } = useWorldById({ worldId: event.worldId });
+
+  const { push: openUrlUsingRouter } = useHistory();
+
+  const handleEnterSpace = useCallback(() => {
+    enterSpace(world?.slug, space?.slug, {
+      customOpenRelativeUrl: openUrlUsingRouter,
+    });
+    onClick();
+  }, [world?.slug, openUrlUsingRouter, onClick, space?.slug]);
+
+  const showDate = Boolean(
+    differenceInCalendarDays(eventEndTime({ event }), eventStartTime({ event }))
+  );
```

Comment:
I believe we should also show date if it's happening not today. Because otherwise, if the event is tomorrow we might think that it's happening today because we show only time.

Reply:
yeah, exactly, this will show the date of the event if it's not today

Reply:
but it checks the difference not with today date, but if there's a difference in dates between end and start time.
But if the event is happening next week but within a day (let's say 1 June 11:00 - 12:00), it'll just show the time, but not date and it'll be confusing to users. 

So there should be 2 checks:
1. if dates of start time and end time are different
2. if they're equal, but the date of the whole event is not today.

Reply:
I've added `!isToday` as an additional check here